depth_image_proc/src/point_cloud_xyz.cpp

```diff
@@ -58,6 +58,9 @@ PointCloudXyzNode::PointCloudXyzNode(const rclcpp::NodeOptions & options)
   // Read parameters
   queue_size_ = this->declare_parameter<int>("queue_size", 5);
 
+  // values used for invalid points for pcd conversion
+  invalid_depth_ = this->declare_parameter<double>("invalid_depth", 0.0);
```

Comment:
for completeness, we should probably add this parameter to the xyzrgb and xyzi components. This new parameter also needs to be added to the documentation for each of the components - https://github.com/ros-perception/image_pipeline/blob/rolling/depth_image_proc/doc/components.rst